src/test/java/uk/gov/hmcts/ccd/data/casedetails/query/CaseDetailsQueryBuilderFactoryTest.java

```diff
@@ -67,7 +67,7 @@ void setUp() {
         @Test
         @DisplayName("should secure new builder instance with user authorisation")
```

Comment:
we need to add unite tests here for the unsecured case we introduced

Reply:
ok 👍

Reply:
DONE